wandb/sdk/wandb_init.py

```diff
@@ -560,6 +566,7 @@ def init(
         # Need to build delay into this sentry capture because our exit hooks
         # mess with sentry's ability to send out errors before the program ends.
         sentry_exc(e, delay=True)
-        raise_from(Exception("problem"), e)
+        reraise(*sys.exc_info())
+        #  raise_from(Exception("problem"), e)
```

Comment:
This shows us what actually happened under the hood instead of the oh so helpful "problem" exception.  I could see us hiding our internal errors, but they're going to be really hard to debug unless someone can get us the logs which are now non-trivial to fine...

Reply:
good